lms/djangoapps/commerce/utils.py

```diff
@@ -44,6 +44,13 @@ class EcommerceService(object):
     def __init__(self):
         self.config = CommerceConfiguration.current()
 
+    def toggle_commerce_config(self, enabled=False, checkout_page='/test_basket'):
+        """ Enable / Disable CommerceConfiguration model """
+        CommerceConfiguration.objects.create(
+            checkout_on_ecommerce_service=enabled,
+            single_course_checkout_page=checkout_page
+        )
```

Comment:
Nevermind, I see that ConfigurationModels keep their history in the same table. Strange how that works, but what you have here makes sense now. I would still update the name of the function since you are not necessarily toggling anything.